allure-support/src/main/kotlin/com/kaspersky/components/alluresupport/interceptors/step/AllureMapperStepInterceptor.kt

```diff
@@ -10,7 +10,7 @@ import java.util.UUID
 
 class AllureMapperStepInterceptor : StepWatcherInterceptor {
 
-    private val lifecycle = AllureAndroidLifecycle
+    private val lifecycle = AllureAndroidLifecycle()
```

Comment:
looks good
let's change the version of the allure library in libs.versions.toml file